mypy/moduleinfo.py

```diff
@@ -21,198 +21,205 @@
 
 third_party_modules = {
     # From Python 3 Wall of Superpowers (https://python3wos.appspot.com/)
-    'simplejson',
-    'requests',
-    'six',
     'pip',
-    'virtualenv',
-    'boto',
-    'setuptools',
-    'dateutil',
-    'pytz',
+    'urllib3',
+    'six',
+    'botocore',
+    'python-dateutil',
```

Comment:
This is the PyPI distribution name, but the actual package name is different, and the package name is required in this file. In this case the package name is `dateutil`. Unfortunately, some manual tweaking seems to be required to get the package name to import.